specification/developerhub/resource-manager/Microsoft.DevHub/preview/2022-04-01-preview/workflow.json

```diff
@@ -854,8 +854,7 @@
       "in": "query",
       "required": false,
       "type": "string",
-      "description": "The ManagedCluster resource associated with the workflows.",
-      "x-ms-parameter-location": "client"
+      "description": "The ManagedCluster resource associated with the workflows."
```

Comment:
I think we should add "x-ms-parameter-location": "method" not removing it.  The default value here is client.

Reply:
Python default location as `method`. I think js has same behavior with python?